app/riot/src/client/preview/render-riot.test.ts

```diff
@@ -23,18 +23,22 @@ beforeEach(() => {
 
 describe('render a riot element', () => {
   it('should not work with nothing', () => {
+    // `render` should only have 1 argument but in every test, there is a magic context given too… 🤷
```

Comment:
I think the 2nd argument can be deleted without any problem.

I think the test has been missed with this commit 
https://github.com/storybookjs/storybook/commit/f555ce0d6956a8fcb9cfd7d9c0fbe899de401bdc#diff-dc99c8ceca53cf99aca341cfaa97633689b0d7edbdaa26631026779cd0e723dc

Reply:
Thanks, you're right, I thought I tried to remove them and everything was OK but I just retried, all tests are 🟢 . 🙇🏻